### PR TITLE
Warm start

### PR DESCRIPTION
**Pull request by mxndrwgrdnr**
- major clean up of run.py
- warm start mandatory activities so UrbanSim HLCM can use commute params
- run UrbanSim container with updated HLCM configs
- run ActivitySim container with warm start capability
- misc. pre/post-processing changes: cache block-to-taz mappings, schools/colleges tables, etc.

### CODE REVIEW

**Comment by mxndrwgrdnr** [comment]
Open question: do we want to include auto-ownership in the warm start updates?

File: settings.yaml [hunk 1]
```diff
@@ -5,30 +5,30 @@
...
+  activitysim: mxndrwgrdnr/activitysim:latest
   polaris:
 docker_stdout: False
 pull_latest: False
 
-# misc pilates settings
-skims_fname: campo_skims.hdf5
-# skims_fname: 10.activitySimODSkims.UrbanSim.TAZ.Full.csv.gz
+# skim settings
+skims_zone_type: taz  # one of [taz, block_group]
```
Comment (jdcaicedo251) [inline settings.yaml:30]: zone type can be different for different regions

**Review by jdcaicedo251 — APPROVED** [review]
Everything looks good to me. No major comments on this PR. The one comment I have is that the skims_zone_type could be different for different regions (at least that happens now). I can solve that in the next PR.

File: settings.yaml (hunk 1 above)
Comment (mxndrwgrdnr) [reply settings.yaml:30]: Right. That's why I have it in the settings.yaml, which the place where users are supposed to make region-specific changes.

**Pull request merged**